data/human/human missions.txt

```diff
@@ -7411,3 +7411,1825 @@ mission "Adelita 5"
+			`	He stops and grins at the camera for a moment. "And that chance, it really has made all the difference. It's been a year of hard work, but I've saved some money and made some investments, and things really are turning around."`
+			`	"By way of thanks," he continues, "I've included 50,000 credits with this message. That should cover the cost of transporting to me to Rand, but it doesn't come close to what I really owe. Which is, in all honesty, my very life."`
+			`	He pauses, as if he's trying to remember something. "Anyways, thank you again, and know I'll never forget what you did for me."`
+			`	Timothy waves and then the video ends.`
+				decline
+
+
+event "timothy radrickson: governor"
+
+
+mission "Timothy Radrickson 3: Governor Notification"
+	name "Visit the Governor on <planet>"
+	description "The governor of <planet> has invited you to speak with them."
+	to offer
+		has "event: timothy radrickson: governor"
+		has "main plot completed"
+	destination "Rand"
+	landing
+	on offer
+		conversation
+			`An indicator light is blinking on your main panel. There is a cached message waiting for you to play.`
+			choice
+				`	(Play the message.)`
+			`	A young woman in very professional clothing appears on the screen. "Captain <last>, I am pleased to inform you that the governor of Rand requests your presence. Please come to Rand at your earliest convenience.`
+
+	on complete
+		conversation
+			`Upon landing you are met by a squat, heavy-set woman in her late 40s. She reaches out a calloused hand to shake yours. "Welcome to Rand, Captain <last>. It is a pleasure, and honor, to have someone of your status here on our little world. I am Regina, the Mayor of the spaceport town, and one of the major company-bosses here. I've been instructed to escort you to the Governor's mansion. I have a vehicle waiting for you just outside the spaceport. Meet me there when you're ready."`
+
+
+mission "Timothy Radrickson 3a: Rescue the Convoy"
+	name "Rescue <origin> convoy"
+	description "<origin>'s largest ever convoy was intercepted on its way to Tarazed. Merchants report that a large pirate fleet escorting a freighter convoy passed by Lesath, and it looks like they went to <destination>. Governor Tim has asked you to try and save it before the pirates can plunder it completely."
+	source "Rand"
+	destination "Greenrock"
+	deadline 7
+	clearance
+	to offer
+		has "Timothy Radrickson 3: Invitation to Rand: done"
+	on offer
+		conversation
+			`Regina, her driver, and a black sleek vehicle are all waiting for you outside the worn down and substandard spaceport. You slide into the comfortable seating and look out the window as the dry, ugly landscape of Rand passes you by. From time to time you see a strip mine or a camp of workers' barracks, but rarely anything else. Behind your vehicle there is a huge plume of half-frozen dust. Apparently low gravity and dust is a terrible combination.`
```

Comment:
So - I agree it would be cool. My understanding is to accomplish this I would need to create branches for every time Rand's environment is discussed and have two different descriptions for each. 

I am not really good at branches and I'd need to see if I could get Bene back to help me with that. And.. it may be a decent amount of work. 

So my question is this: Is Rand terraformed enough by the missions that it deserves two very different descriptions or is it a more gradual/incomplete change where at the time the differences wouldn't be pronounced enough yet for the player to clearly experience them?